blocks/simple-list-block/features/simple-list/simple-list.scss

```diff
@@ -45,13 +45,19 @@
       padding-left: calculateRem(16px);
       text-decoration: none;
 
-      .simple-list-headline-text {
+      .simple-list-headline-text { /* stylelint-disable-line */
         font-size: calculateRem(16px);
         font-weight: normal;
         line-height: calculateRem(24px);
-        max-height: 72px;
         overflow: hidden;
+        /*
```

Comment:
Even though they are multiline comments, should be use multiple single-line comments w/ `//`?
Assuming those will be stripped out when compiled, making the CSS smaller while preserving the documentation internally?